pubspec.yaml

```diff
@@ -146,6 +145,10 @@ dependencies:
       url: https://github.com/maRci002/video_thumbnail.git
       ref: feat-web_implementation
       path: video_thumbnail
+  media_kit: ^1.1.7
+  media_kit_video: ^1.1.8
+  media_kit_libs_video: ^1.0.1
+  video_player: ^2.7.2
```

Comment:
Use all libs ?

Reply:
yes, some place still needs to use video_player